lib/collection.js

```diff
@@ -377,7 +379,6 @@ Collection.prototype.find = deprecateOptions(
             ? normalizeHintField(options.hint)
             : this.s.collectionHint,
         noCursorTimeout: options ? options.timeout : null,
-        awaitData: options ? options.awaitdata : null,
```

Comment:
It would send `awaitData` with an undefined value if `options.awaitdata` was undefined so this shouldn't have any effect server-side. This brings up the question of whether we should be using `options.awaitData` or `options.awaitdata`. Both are present in `cursor_tests`.